content/_index.md

```diff
@@ -3,34 +3,23 @@ title = "Stephen Augustus"
 tagline = "Caesar of Systems"
 +++
 
-Stephen is a Black engineering director and leader in open source communities.
+Stephen is a Black engineering director and leader in open source communities, based in New York City.
 
 He is the Head of [Open Source at Cisco][cisco-open-source], working within the
-Strategy, Incubation, & Applications (SIA) organization.
+Corporate Strategy, Development, and Incubation (CSDI) organization.
 
-For [Kubernetes][kubernetes], he has co-founded transformational elements of
-the project, including the [KEP][enhancements] (Kubernetes Enhancements
-Proposal) process, the [Release Engineering][releng] subproject, and Working
-Group Naming. Stephen has also previously served as a chair for both SIG PM and
-SIG Azure.
+Outside of managing Cisco's open source strategy and a team of brilliant architects in Cisco's Open Source Program Office (OSPO), Stephen participates in foundation-level governing boards (OpenSSF, CNCF, OpenAPI Initiative, project-level steering committees (Kubernetes, OpenSSF Scorecard, TODO Group), and helps maintain a few codebases around open source projects you may have heard of.
 
-He continues his work in Kubernetes as a [Steering Committee][k8s-steering]
-member and a Chair for [SIG Release][sig-release].
+Additionally, he serves as an advisor and invests in startups with open source in their DNA.
```

Comment:
how about: Additionally, he serves as a startup advisor and avid investor in young companies with open source in their DNA.

Reply:
Fixed in https://github.com/justaugustus/about/pull/11/commits/ccb73c31273a02c7d5f68108ae49be8dd85b47a7.